src/contexts/Api/index.tsx

```diff
@@ -174,6 +174,8 @@ export const APIProvider = ({ children }: { children: React.ReactNode }) => {
     } else {
       _provider = new WsProvider(endpoints.rpc);
     }
+    provider?.disconnect(); // Disconnect the previous provider - no need to wait for it, can be done in the background
```

Comment:
Missing disconnect wasn't causing any bugs, but it held the connection open and, in case of absent servers under certain addresses, would spam console with connection errors, trying to reconnect every couple of seconds.